packages/embeds/embed-core/playground.ts

```diff
@@ -1,14 +1,13 @@
-import type { GlobalCal, GlobalCalWithoutNs } from "./src/embed";
+import type { GlobalCal } from "./src/embed";
 
-type A = GlobalCalWithoutNs;
 const Cal = window.Cal as GlobalCal;
 const callback = function (e) {
   const detail = e.detail;
   console.log("Event: ", e.type, detail);
 };
 
 const searchParams = new URL(document.URL).searchParams;
-const only = window.only;
+const only = searchParams.get("only");
```

Comment:
Get only param from here directly instead of relying on window.only which is set by playground index.html